smac/facade/epils_facade.py

```diff
@@ -339,6 +338,7 @@ def _get_rng(rng):
 
         # initialize random number generator
         if rng is None:
+            self.logger.info('no rng given, falling back to non-deterministic behaviour')
```

Comment:
This shouldn't be a info, but rather a debugging message

Reply:
I would consider this a warning that should not occur in the best case.

However, I don't get why we fall back to non-deterministic behavior. I would expect that the initialize our own rng in this case

Reply:
Using a default seed of 1 now.